### DISCUSSION TITLE
Any way to disable "The React Flow parent container needs a width and a height" warning?

### DISCUSSION THREAD

**Discussion by elringus** [xyflow/xyflow#2927] [Q&A] [answered]
The warning fires false-positive when the flow is rendered inside a tab (https://github.com/caplin/FlexLayout) when switching to another tab. Is there any way to suppress the warning?

```
[React Flow]: The React Flow parent container needs a width and a height to render the graph. Help: https://reactflow.dev/error#004
```

**Comment by bcakmakoglu** [comment, marked as answer]
The warnings aren’t shown in production if I’m not wrong 
Other than that it’s possible to handle errors and warning yourself using the onError prop :)

**Reply by elringus** [reply]
Ah, so the warnings are handled with onError as well; should've checked before asking. Thanks!

**Comment by alexandreczg** [comment]
@elringus How did you resolve this with the `onError` ?

I added the following but still getting the pesky warnings. 

```tsx
 <ReactFlow
    nodes={nodes}
    edges={edges}
    onEdgeMouseEnter={handleEdgeMouseEnter}
    onEdgeMouseLeave={() => setHoveredEdge(null)}
    onEdgeClick={handleEdgeMouseClick}
    nodeTypes={nodeTypes}
    panOnDrag
    fitView
    defaultEdgeOptions={defaultEdgeOptions}
    zoomOnDoubleClick={false}
    onError={(error) => console.log(error)}
>
```

I also have mine on a `tab` on a modal.

**Reply by elringus** [reply]
I have this:

```ts
<ReactFlow onError={handleError} ...

function handleError(code: string, message: string) {
    if (code === "004") return; // False-positive when changing tabs.
    console.warn(message);
}
```

Though I'm still on `11.10.1`, something could've changed in later versions.

**Comment by alexandreczg** [comment]
Doesn't work for me :/, still throwing the warning.

**Comment by gerbil** [comment]
Got the same problem here..
